backend/tests/unit/dal/Services/ParcelServiceTest.cs

```diff
@@ -129,6 +219,16 @@ public void Get()
             Assert.NotNull(result);
             Assert.Equal(EntityState.Detached, context.Entry(result).State);
             Assert.Equal(parcel, result, new ShallowPropertyCompare());
+            Assert.NotNull(parcel.Address);
```

Comment:
Started confirming the query result to ensure `Include(...)` was implemented.  Not 100% sure InMemoryDatabase will do this correctly.